src/spoke/libraries/UpdateContractMessageLib.sol

```diff
@@ -64,4 +66,53 @@ library UpdateContractMessageLib {
     function serialize(UpdateContractSyncDepositMaxReserve memory t) internal pure returns (bytes memory) {
         return abi.encodePacked(UpdateContractType.SyncDepositMaxReserve, t.assetId, t.maxReserve);
     }
+
+    //---------------------------------------
+    //   UpdateContract.Toggle (submsg)
```

Comment:
Idea is to try to design these as generic, reusable update contract messages, that serve multiple purposes